plugins/modules/azure_rm_virtualmachine.py

```diff
@@ -203,6 +203,15 @@
             - ReadWrite
         aliases:
             - disk_caching
+    os_disk_encryption_set:
+        description:
+            - Disk encryption set for the OS disk.
+        type: dict
```

Comment:
Consistent with the definition of arg
```suggestion
        type: raw
```

Reply:
Updated. Thanks.